xml/depl_nodes.xml

```diff
@@ -163,10 +163,7 @@
     create and propagate zones and records over the network using pools of DNS
     servers. Deployment defaults are in place, so not much is required to
     configure &desig;. &o_netw; needs additional settings for integration with
-    &desig;. The &desig; barclamp does not configure these &o_netw; settings; an
-    admin can enable them if necessary. For more information, see the <link
-    xlink:href="https://docs.openstack.org/designate/latest/contributor/integrations.html">OpenStack
-    Integrations</link>.
+    &desig;, which are also present in the '[designate]' section in &o_netw; configuration.
```

Comment:
better to use tags for designate...in the <literal>[designate]</literal section

Reply:
Done, in next commit.

Reply:
`<literal>[designate]</literal>`